spec/rubocop/cop/metrics/line_length_spec.rb

```diff
@@ -21,6 +21,14 @@
     expect_no_offenses('#' * 80)
   end
 
+  it 'accepts the first line is the shebang line' do
```

Comment:
Maybe `accepts the first line if it is a shebang line` sound better.

Reply:
Thanks for the review. I fixed it.